GovUk.Frontend.Umbraco.Testing/UmbracoDictionaryFactory.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Moq;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Services;
+
+namespace GovUk.Frontend.Umbraco.Testing
+{
+    public static class UmbracoDictionaryFactory
```

Comment:
To be consistent with [similar existing functionality](https://github.com/thepensionsregulator/govuk-frontend-aspnetcore-extensions/blob/develop/GovUk.Frontend.Umbraco.Testing/UmbracoPublishedElementExtensions.cs) I would expect:

- the class to be called `LocalizationServiceExtensions`
- the method to be called `SetupDictionaryItem` or `SetupUmbracoDictionaryItem` ([Item is Umbraco terminology](https://docs.umbraco.com/umbraco-cms/fundamentals/data/dictionary-items/))
- the method should return the mock, to allow a fluent interface
- the method should have two overloads to accept either the mock or the mocked object